prime-angular-frontend/src/app/modules/enrolment/pages/pharmanet-enrolment-summary/pharmanet-enrolment-summary.component.html

```diff
@@ -8,59 +8,133 @@
-
-            <button mat-flat-button
+        <form [formGroup]="form">
+
+          <ng-container *ngFor="let careSetting of careSettings">
+
+            <ng-container [ngSwitch]="careSetting.careSettingCode">
+
+              <ng-container *ngSwitchCase="CareSettingEnum.PRIVATE_COMMUNITY_HEALTH_PRACTICE"
+                            [ngTemplateOutlet]="settingEmail"
+                            [ngTemplateOutletContext]="{
+                              setting: 'Private Community Health Practice',
+                              settingCode: careSetting.careSettingCode,
+                              formControlName: 'communityHealthEmails',
+                              formControl: communityHealthEmails
+                            }">
+              </ng-container>
+
+              <ng-container *ngSwitchCase="CareSettingEnum.COMMUNITY_PHARMACIST"
+                            [ngTemplateOutlet]="settingEmail"
+                            [ngTemplateOutletContext]="{
+                              setting: 'Community Pharmacy',
+                              settingCode: careSetting.careSettingCode,
+                              formControlName: 'pharmacistEmails',
+                              formControl: pharmacistEmails
+                            }">
+              </ng-container>
+
+              <ng-container *ngSwitchCase="CareSettingEnum.HEALTH_AUTHORITY"
+                            [ngTemplateOutlet]="settingEmail"
+                            [ngTemplateOutletContext]="{
+                              setting: 'Health Authority',
+                              settingCode: careSetting.careSettingCode,
+                              formControlName: 'healthAuthorityEmails',
+                              formControl: healthAuthorityEmails
+                            }">
+              </ng-container>
+
+            </ng-container>
+
+          </ng-container>
+        </form>
+
+        <ng-template #settingEmail
+                     let-setting="setting"
+                     let-settingCode="settingCode"
+                     let-formControlName="formControlName"
+                     let-formControl="formControl">
+          <app-page-subheader headerType="h6">
+            <ng-container appPageSubheaderTitle>
+              {{ setting }}
+            </ng-container>
+            <ng-container appPageSubheaderSummary>
+              Send your approval to your private community health practice's PharmaNet administrator (e.g., office
+              manager).
+            </ng-container>
+
+          </app-page-subheader>
+
+          <form [formGroup]="form">
+
+            <div class="row"
+                 [hidden]="!isEmailVisible(settingCode)">
+              <div class="col-sm-8 col-xl-7">
+
+                <mat-form-field class="w-100 mb-3">
+                  <input matInput
+                         placeholder="Email"
+                         [formControlName]="formControlName">
```

Comment:
You should be able to change this to `[formControl]="formControl"` and drop the `formControlName` from your passed bindings to the `ng-template`